File: app/src/main/java/com/moggot/findmycarlocation/about/PrivacyPolicyActivity.java
package com.moggot.findmycarlocation.about;

import android.annotation.SuppressLint;
import android.os.Bundle;
import android.webkit.WebSettings;
import android.webkit.WebView;
import android.webkit.WebViewClient;

import androidx.annotation.Nullable;

import com.moggot.findmycarlocation.R;
import com.moggot.findmycarlocation.base.BaseActivity;

public class PrivacyPolicyActivity extends BaseActivity {

    WebView webView;

    @SuppressLint("SetJavaScriptEnabled")
    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_privacy_policy);
        webView = findViewById(R.id.privacy_policy_webview);

        String url = getString(R.string.privacy_policy_link);
        webView.setWebViewClient(new WebViewClient());
        WebSettings webSettings = webView.getSettings();
        webSettings.setJavaScriptEnabled(true);
        webView.loadUrl(url);
    }

    @Override
    public int getFragmentContainerId() {
        return 0;
    }
}

File: app/src/main/java/com/moggot/findmycarlocation/about/PrivacyPolicyActivity.kt
package com.moggot.findmycarlocation.about

import android.os.Bundle
import android.webkit.WebSettings
import android.webkit.WebViewClient
import com.moggot.findmycarlocation.R
import com.moggot.findmycarlocation.common.BaseActivity
import com.moggot.findmycarlocation.databinding.ActivityPrivacyPolicyBinding
import dagger.hilt.android.AndroidEntryPoint

@AndroidEntryPoint
class PrivacyPolicyActivity : BaseActivity<ActivityPrivacyPolicyBinding>() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        val url: String = getString(R.string.privacy_policy_link)
        viewBinding.privacyPolicyWebview.webViewClient = WebViewClient()
        val webSettings: WebSettings = viewBinding.privacyPolicyWebview.settings
        webSettings.javaScriptEnabled = true
        viewBinding.privacyPolicyWebview.loadUrl(url)
    }

    override fun performDataBinding(): ActivityPrivacyPolicyBinding {
        return ActivityPrivacyPolicyBinding.inflate(layoutInflater)
    }
}
